src/Workspaces/Core/Portable/Differencing/LongestCommonSubsequence.cs

```diff
@@ -375,13 +416,11 @@ protected double ComputeDistance(TSequence oldSequence, int oldLength, TSequence
         /// 
         /// VArrays store just the y index because x can be calculated: x = y + k.
         /// </remarks>
-        private VStack ComputeEditPaths(TSequence oldSequence, int oldLength, TSequence newSequence, int newLength)
+        private void ComputeEditPaths(ref VStack stack, TSequence oldSequence, int oldLength, TSequence newSequence, int newLength)
```

Comment:
Why the change to ref here? All places are just creating right before calling anyways

Reply:
I'll undo. It made more sense before I made subsequent changes.